mailgun/route_handlers.py

```diff
@@ -24,9 +27,26 @@
 listserv_client = ListservClient()
 
 
+def log_and_report_errors():
```

Comment:
Blegh...this feels like not quite the right name, but I don't have a better one.